contracts/protocol/facets/MetaTransactionsHandlerFacet.sol

```diff
@@ -31,6 +31,7 @@ contract MetaTransactionsHandlerFacet is IBosonMetaTransactionsHandler, Protocol
 
         // Set input type for the function name
         pmti.inputType[COMMIT_TO_OFFER] = MetaTxInputType.CommitToOffer;
+        pmti.inputType[COMMIT_TO_CONDITIONAL_OFFER] = MetaTxInputType.CommitToOffer;
```

Comment:
A new `MetaTxInputType` is needed for `COMMIT_TO_CONDITIONAL_OFFER`.

`MetaTxInputType.CommitToOffer` does not include `tokenId` parameter, so the buyer would be signing only `buyer` and `offerId`. Therefore the relayer can choose an arbitrary `tokenId` and the metaTx will succeed (as long as the buyer actually owns it). We need `MetaTxInputType.ConditionalCommitToOffer` which includes the `tokenId`.

To fully support it, we also need a hashing method `hashConditionalOfferDetails` which will correctly hash the inputs.

Reply:
ok